deploy/template.yaml

```diff
@@ -228,6 +228,16 @@ Conditions:
     - Condition: DeployAlarms
     - Condition: ApplyReservedConcurrency
   PerformanceTestEnv: !Equals [ !Ref Environment, build]
+  NotLocalTestStack:
```

Comment:
Could we maybe add a line to the README to say that you can't use custom domains in custom stacks and you should use the main stack if you need this?